.azuredevops/deploy-platform-core-fluxnotification.yaml

```diff
@@ -12,6 +12,10 @@ parameters:
     values:
       - All
       - Shared Event Hub
+      - FluxNotificationApi - All
```

Comment:
Do we need 4? Could we just have Managed Identities and that covers all? would simplify logic

Reply:
@DaRoszko : I have done the changes